collect_app/src/main/java/org/odk/collect/android/utilities/AppStateProvider.kt

```diff
@@ -5,18 +5,17 @@ import org.odk.collect.android.preferences.keys.MetaKeys
 import org.odk.collect.shared.Settings
 
 class AppStateProvider(private val packageInfo: PackageInfo, private val metaSettings: Settings) {
+
     fun isFreshInstall(): Boolean {
-        return alwaysFresh.let {
```

Comment:
This would cause problems if a test returns to the splash screen. We just want to fake that the app is not being upgraded.